src/pages/workspaces/workspace/Dashboard/WorkspaceInformation.test.ts

```diff
@@ -99,4 +101,43 @@ describe('WorkspaceInformation', () => {
     // Accessibility
     expect(await axe(container)).toHaveNoViolations();
   });
+
+  it.each([
+    { policies: [] },
+    {
+      policies: [
+        {
+          namespace: 'terra',
+          name: 'group-constraint',
+          additionalData: [{ group: 'foo' }],
```

Comment:
Until we know that multiple `group-constraint` policies are not possible, I would suggest you change this so that the group name is different, and then make sure that your message shows all 3 group names.

Reply:
I reused a group name in both policies because I wanted to test that the message de-duplicated group names.

Reply:
Then can you put a second, different group in this one? In addition to testing that it dedupes, it would be nice to know that it actually pulls the group names from both (instead of only respecting the second group policy, for example).

Reply:
I removed the second group here and added a test for `groupConstraintMessage` to test de-duplication in https://github.com/DataBiosphere/terra-ui/pull/4469/commits/34ed68d9e54ee0ff8b0af44e9e46325c2eda55a3.